vmms/ec2SSH.py

```diff
@@ -380,3 +380,9 @@ def existsVM(self, vm):
             if inst.instances[0].id is vm.ec2_id:
                 return True
         return False
+
+    def getImages(self):
+        """ getImages - Lists all images in TASHI_IMAGE_PATH that have the
+        .img extension
+        """
+        return ["test.img"]
```

Comment:
Sorry about this hacking, but empty list wouldn't work. The list returned by getImages is used for validation, i.e. whether the image specified by the user is in the list. Also, Tango will do some other works regarding this name.

After a second thought, I believe that test.img is badly named. Perhaps default.img is better, which means using the default AMI specified in the config.py file. (Perhaps the corresponding wiki pages https://github.com/autolab/Tango/wiki/Testing-Tango and https://github.com/autolab/Tango/wiki/Set-up-Amazon-EC2-VMMS should change as well.) 

FYI, an alternative is to use a bypass list and change all references to the function getImages. But that means for every future service provider, we should update that list.